website/versioned_docs/version-6.x/generator.md

```diff
@@ -1,5 +1,4 @@
 ---
-# Don't edit this file directly, it was copied using scripts/download-readmes.js: 
```

Comment:
I guess these are versioned/ anyway so no need to do have anything here?

Reply:
Yes, so I removed the comment and only left the stuff required by docusaurus like id, original_id etc. Or do you mean we should leave the comment there as well?

Reply:
The purpose of the comments are just so people don't try to modify the files that are imported from other repos. However because if this issue we were thinking we should just move all the docs to this repo and just link to the website instead as the readmes

Reply:
That would certainly make things easier. The way it currently is can also work though. Current version in /docs dynamic with the "don't edit" comment, past versions in /versioned committed to this repo. A script can be written to strip the comments in /versioned after a new version is cut. A little more work, but maybe worth it to avoid that extra click from the npm readme page.

Reply:
yeah the issue is that the linking and everything just doesn't work when the file is generated so it seems like it makes more sense to have generic information in the readmes in the babel/babel repo and then link to the "real" docs on this repo instead.